src/main/kotlin/no/nav/familie/ef/mottak/task/SendDokumentasjonsbehovMeldingTilDittNavTask.kt

```diff
@@ -49,37 +64,34 @@ class SendDokumentasjonsbehovMeldingTilDittNavTask(
         }
     }
 
-    private fun lagLinkMelding(søknad: Søknad, dokumentasjonsbehov: List<Dokumentasjonsbehov>): LinkMelding {
+    private fun opprettSendPåminnelseTask(task: Task) {
+        taskService.save(
+            Task(
+                SendPåminnelseOmDokumentasjonsbehovTilDittNavTask.TYPE,
+                task.payload,
+                Properties(task.metadata).apply {
+                    this["eventId"] = UUID.randomUUID().toString()
+                }
+            ).medTriggerTid(VirkedagerProvider.nesteVirkedag(LocalDate.now().plusDays(2)).atTime(10, 0))
+        )
+    }
```

Comment:
Er dette noe som man kan flytte til `SendPåminnelseOmDokumentasjonsbehovTilDittNavTask`, eks i

```kotlin
companion object {
 fun opprettTask(søknadId: UUID): Task {
    return Task(
                SendPåminnelseOmDokumentasjonsbehovTilDittNavTask.TYPE,
                søknadId.toString(),
                Properties(task.metadata).apply {
                    this["eventId"] = UUID.randomUUID().toString()
                }
            ).medTriggerTid(VirkedagerProvider.nesteVirkedag(LocalDate.now().plusDays(2)).atTime(10, 0))
 }
}
```

Reply:
Hva er `eventId` ?

Reply:
Tror det er påkrevd i meldingen til ditt nav for at vi/de skal kunne "spore" den senere.

Reply:
Godt forslag om å trekke det ut til egen opprettTask i riktig klasse.